build.zig

```diff
@@ -131,6 +133,7 @@ pub const LibSokolOptions = struct {
     backend: SokolBackend = .auto,
     use_egl: bool = false,
     use_x11: bool = true,
+    dylib: bool = false,
```

Comment:
I wonder if people outside the Apple ecosystem know what a `dylib` is, but I don't have an immediately better suggestion (except maybe `dynamic_linkage` because it matches the Zig build system lingo).